Xamarin.Essentials/Sms/Sms.mac.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using AppKit;
+using Foundation;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Sms
+    {
+        internal static bool IsComposeSupported =>
+            MainThread.InvokeOnMainThread(() => NSWorkspace.SharedWorkspace.UrlForApplication(NSUrl.FromString("sms:")) != null);
+
+        static Task PlatformComposeAsync(SmsMessage message)
+        {
+            var firstNumber = message.Recipients?.FirstOrDefault();
```

Comment:
what happens if there's more than one recipient ?

Reply:
Added support for mutliple, thanks :)